package.json

```diff
@@ -48,7 +46,6 @@
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-standard": "^4.0.0",
     "jasmine": "^3.3.1",
-    "npm-run-all": "^4.1.5",
```

Comment:
Why get rid of npm-run-all?

Reply:
I didn't think it was necessary to keep for only one use case.

There were two cases, but that second case was not actually needed. 

IF we still had a pattern matching requirement and didn't require any order, maybe it would have been worth keeping. 

In the end, I felt it became more of bloatware since there was only one use case left and all it was doing was shorting the line by 13 chars.

```
run-s eslint unit-tests java-unit-tests e2e-tests
```
and
```
npm run eslint && npm run cover && npm run java-unit-tests
```

The package itself is 156kb in size but with all its dependencies it was about 2.4mb. Even though some of the sub-dependencies we may already have from other packages.

I felt it might be better to just cut/downsize.

Reply:
For `dependencies` I would be strongly in favor of this kind of a reduction. But for `devDependencies` which is build tooling I do not really see the benefit of getting rid of `npm-run-all`. Even if we would only use it once I can imagine it would be nice if we could use it in more tasks someday.

The main thing is that I personally find it nicer if we don't have to do a lot of shell things in `package.json` itself.

But I don't feel super strongly about one way or the other, leaving it up to your best judgement.